git/config/install

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+# prototype-kicad-library-ci
+
+# File: git/config/install
+# Description: Git config entries install script for the prototype-kicad-library-ci
+#   repository.
+
+function error()
+{
+    local -r message="$1"
+
+    ( >&2 echo "$mnemonic: $message" )
+}
+
+function abort()
+{
+    if [[ "$#" -gt 0 ]]; then
+        local -r message="$1"
+
+        error "$message, aborting"
+    fi
+
+    exit 1
+}
+
+function display_help_text()
+{
+    echo "NAME"
+    echo "    $mnemonic - Install the prototype-kicad-library-ci repository's Git config."
```

Comment:
Please change `Git config` to `Git config entries` for consistency with the script comment block, and updated README instructions.
```suggestion
    echo "    $mnemonic - Install the prototype-kicad-library-ci repository's Git config"
    echo "        entries."
```